src/penalty.h

```diff
@@ -191,6 +200,30 @@ namespace Faunus {
                             };
                         }
 
+                        else if (property=="L/R") {
```

Comment:
document

Reply:
Thanks, Mikael! The docs are now updated.